### PR TITLE
docs: add docs for backup COS

### PR DESCRIPTION
**Pull request by maudL1n**
1. add document section for root folder README file
2. add README file for docs folder
3. add COS backup guide

### CODE REVIEW

**Review by tangcong — CHANGES_REQUESTED** [review]
thanks. The picture url seems to have a problem and cannot be displayed. (two docs)
https://github.com/maudL1n/kstone/blob/docs/docs/backup/..%5Cimages%5Cimages_for_backup_cos%5Ckstone-associate.png

File: docs/backup/Backup guide for Tencent COS_zh.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+
+内存大小:单个节点的内存, 单位: GiB
+
+磁盘类型: 磁盘种类, 比如CLOUD_SSD/CLOUD_PREMIUM/CLOUD_BASIC
+
+磁盘大小: 单个节点的磁盘大小, 单位: GB
+
+集群规模: etcd集群成员数量: 支持1, 3, 5, 7
+
+集群节点映射: etcd集群的内网还有外网地址, 如果不用刻意省略
```
Comment (neweyes) [inline docs/backup/Backup guide for Tencent COS_zh.md:45]: 刻意 -》 可以

File: docs/backup/Backup guide for Tencent COS_zh.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+  - 点击 "集群功能项"
+
+  - 点击 "Backup:"
+
+  - 填好下面的参数
+
+  - BackupIntervalInSecond 决定了多久backup-operator会进行一次etcd备份工作
+
+  - MaxBackups 是你想要存放的etcd备份文件的最大数量
```
Comment (neweyes) [inline docs/backup/Backup guide for Tencent COS_zh.md:80]: 超过这个数量将会怎么样

Comment (tangcong) [reply docs/backup/Backup guide for Tencent COS_zh.md:80]: 会自动清理

**Review by tangcong — APPROVED** [review]
good job. lgtm.

**Pull request merged**